build_windows.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
@hobbes1069 @ac0zj I wrote this script to automate the windows X-compile.  Does this work for you guys?  If so, can I modify the README.md to use this script?